jina/peapods/zmq.py

```diff
@@ -187,10 +186,6 @@ def print_stats(self):
                          f'#recv: {self.msg_recv} '
                          f'sent_size: {get_readable_size(self.bytes_sent)} '
                          f'recv_size: {get_readable_size(self.bytes_recv)}')
-        profile_logger.debug({'msg_sent': self.msg_sent,
-                              'msg_recv': self.msg_recv,
-                              'bytes_sent': self.bytes_sent,
-                              'bytes_recv': self.bytes_recv})
```

Comment:
note, removing this means dashboard lose all info in `TASK` panel

Reply:
if the default logger has the profile handler?

Reply:
when a logger can have multiple handlers, why to have an specific logger for Profile?

Reply:
i don't remember the details, but it could be due to the easy toggle on/off, or `profile_logger` relies on some multi-thread/processing`Event`

Reply:
Profile Logger is also using a `QueueHandler`, so it behaves as the sse_logger from behavior point of view

Reply:
I will add a dedicated logger for PROFILE to avoid it getting every other emition